test/tools/libutils/FakeEvmc.cpp

```diff
@@ -211,13 +213,14 @@ evmc_context_fn_table const fakeFnTable = {
     log,
 };
 
-FakeEvmc::FakeEvmc(evmc_instance* _instance)
+FakeEvmc::FakeEvmc(evmc_vm* _instance)
 {
     fakeDepth++;
     m_depth = fakeDepth;
     m_instance = _instance;
-    m_context = new evmc_context();
-    m_context->fn_table = &fakeFnTable;
+    m_context = std::make_shared<evmc_host_context>();

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'm_context' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5674743)